CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -952,6 +952,7 @@ local function CreateSettingsHub()
 		local panel = Panel3D.Get(Panel3D.Panels.Settings)
```

Comment:
Hmm, the panels are just named and don't necessarily carry any specific behavior internally.